<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des fournisseurs</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 10pt;
}
#Form{
	
	background-color: rgb(255, 255, 255);
	font: 10pt;
}

.QPushButton{
	border-radius: 2;
	background-color: qlineargradient(spread:pad, x1:0.548, y1:1, x2:0.469455, y2:0, stop:0.627119 rgba(242, 101, 101, 255), stop:1 rgba(255, 146, 117, 255));
	border-bottom: 2px solid;
	border-bottom-color:rgb(221, 83, 83);
	padding: 3px 16px;
	color:rgb(255, 221, 221);	
	font: 10pt;
}
.QPushButton:pressed{
	background-color: rgb(220, 91, 91);
	border-bottom-color: rgb(194, 73, 73);
}

.QLineEdit{
	border-radius: 2;
	border-bottom: 2px solid rgb(166, 166, 166);
	font: 10pt;
	padding: 3px 6px;
	
	background-color:rgb(222, 222, 222);
}

.QComboBox{
	border-radius: 2;
	background-color:rgb(222, 222, 222);
	border: none;
	padding: 3px 6px;
	border-bottom: 2px solid rgb(166, 166, 166);
}
.QComboBox::drop-down{
	 subcontrol-origin: padding;
    subcontrol-position: top right;
	
	border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}
QTableWidget{
	
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="ajouter_button">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="modifier_button">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>357</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="annuler_button">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>annuler_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
